data/blog/2024/2024-07-02-dotkernel-a-replacement-for-laminas-api-tools.md

```diff
@@ -0,0 +1,79 @@
+---
+id: 2024-07-02-dotkernel-a-replacement-for-laminas-api-tools
+author: julian
+title: 'Dotkernel API: a replacement for legacy Laminas API Tools'
+draft: false
+public: true
+created: '2024-07-02T11:00:00-01:00'
+updated: '2024-07-02T11:00:00-01:00'
+tags:
+  - laminas api tools
+---
+
+On Monday, **6 May 2024**, the Technical Steering Committee for the Laminas Project held its monthly meeting to discuss,
+among other things the **Laminas API Tools (formerly Apigility)** alternatives.
```

Comment:
The summary does not match the content, so I recommend an adaptation.

Reply:
I update it a bit, but i am open to suggestions :-)